README.md

```diff
@@ -1,2 +1,183 @@
+
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">VSCode</a></li>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+</details>
+<br/>
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- User authentication
+- Users can add a transaction category
+- Users can add category details
+- Users can sign out
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a id="getting-started"></a>
+<br>
+
+### Prerequisites
+
+In order to reproduce the receipe-app and be able to make changes for your own purpose, you need the following tools:
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [git-scm](https://git-scm.com/)
+- A [Github](https://github.com/) account
+
+### Setup
+
+Clone this repository to your desired folder by running the following command:
+
+```sh
+  git clone https://github.com/skabeo/23-books.git
+```
+
+### Install
+
+To use this application, you need to have `ruby` and `rails gem` installed on your machine. Once you have them installed,
+
+Run bundle install to install all dependencies
+
+```sh
+  cd 23-books
+  bundle install
+
+```
+
+Once you have the files on your machine, you can view it on your browser on `https://localhost:3000` by running
+
+```
+rails server
+```
+
+### Run tests
+
+To run model tests,run the following command:
+```
+rspec spec/models
+```
+To run integration tests,run the following command:
+```
+rspec spec/integrations
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Live demo <a id='live'></a>
+
+To view the live demo visit click [here](https://spec-mytracker.onrender.com/)
+
+## 👥 Author <a id="authors"></a>
+
+👤 **Spencer Okyere**
+
+- GitHub: [@skabeo](https://github.com/skabeo)
+- Twitter: [@black_okyere](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/spencer-okyere)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a id="future-features"></a>
+
+- Infinite scrolling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/skabeo/23-books/issues).
+
+If you would like to contribute to this project, you can follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your changes.
+3. Make your changes and commit them.
+4. Push your changes to your forked repository.
+5. Create a pull request back to the original repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a id="support"></a>
+
+If you like this project, please give it a star on GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
```

Comment:
- [ ] The acknowledgement section should contain an appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).